addons/account/models/account_move.py

```diff
@@ -5144,6 +5192,8 @@ def message_new(self, msg_dict, custom_values=None):
         if (custom_values or {}).get('move_type', 'entry') not in ('out_invoice', 'in_invoice', 'entry'):
             return super().message_new(msg_dict, custom_values=custom_values)
 
+        self = self.with_context(skip_is_manually_modified=True)  # noqa: PLW0642
```

Comment:
I think the context key should only be used to identify flows that are originating from the OCR (I would actually rename the key to be more specific and prevent its use where it should not).
One way we could prevent its use here, is to create a blacklist of fields that should be considered as manually changed, like amount, date, etc... and use both the context key and the blacklist to determine whether to tag a bill as `is_manually_modified`.
What do you think?

Reply:
I do like the idea of a blacklist indeed, I'll check with the AVW what he thinks and which fields we should put in that list.

Reply:
Just finished a call with AVW and for now we would rather be extra safe and use all fields so I'll leave this unchanged for now